### PR TITLE
 project-chatbot

### PR DESCRIPTION
**Pull request by ghost**
https://travelbotjm.netlify.app/

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -6,28 +6,53 @@ body {
...
+  position:relative;
+  width: 100%;
+  height: 100%;
+  font-size: 100px;
+ 
+
+}
+
+
+.container__video-background{
```
Comment (Kaliine) [inline style.css:23]: Such a cool idea to add a video background and the choice you did really fits the subject of the travel bot.

File: style.css [hunk 2]
```diff
@@ -37,39 +62,56 @@ input {
...
   padding: 16px;
   font-size: 16px;
   font-family: 'Montserrat';
   font-weight: 600;
   line-height: 26px;
   flex: 1;
+  margin-top: 5%;
+}
+
+input:focus{
```
Comment (Kaliine) [inline style.css:76]: Very classy effect that you chose here.

File: style.css [hunk 3]
```diff
@@ -37,39 +62,56 @@ input {
...
   max-width: 700px;
   height: 600px;
   border-radius: 30px;
-  background: #fff;
   padding: 20px 24px;
   padding-top: 0;
   box-sizing: border-box;
   display: flex;
   flex-direction: column;
+  backdrop-filter: blur(20px);
```
Comment (Kaliine) [inline style.css:95]: What does this backdrop-filter do?

File: style.css [hunk 4]
```diff
@@ -37,39 +62,56 @@ input {
...
   flex: 1;
   display: flex;
   justify-content: flex-start;
   overflow: scroll;
+  overflow-x: hidden;
   flex-direction: column;
   padding-bottom: 16px;
 }
 
+::-webkit-scrollbar {
```
Comment (Kaliine) [inline style.css:108]: What exactly does this webkit-scrollbar do?

File: style.css [hunk 5]
```diff
@@ -120,21 +163,25 @@ main {
...
   border-radius: 4px;
   padding: 16px 40px;
   margin-right: 4px;
+  margin-top: 5%;
   font-size: 16px;
   line-height: 26px;
   font-family: 'Montserrat';
   font-weight: 500;
   cursor: pointer;
   transition: all 0.3s ease;
```
Comment (Kaliine) [inline style.css:178]: We only see the transition effect when hovering over the button, so what does this transition do to the button (when not hovering)?

File: style.css [hunk 6]
```diff
@@ -120,21 +163,25 @@ main {
...
+  text-shadow:0 0 2px rgb(0, 0, 0);
 }
 
 button:hover {
```
Comment (Kaliine) [inline style.css:182]: Really classy effect you did here.

File: script.js [hunk 7]
```diff
@@ -0,0 +1,201 @@
+// All the DOM selectors stored as short variables
+const chat = document.getElementById("chat");
+const inputWrapper = document.getElementById("input-wrapper");
+const input = document.getElementById("input");
+const sendBtn = document.getElementById("send");
+
+let questionNumber = 1;
+
+const botReply = (msg) => {
```
Comment (Kaliine) [inline script.js:9]: Nice that you went the extra mile and created a bot- and user-reply function. This makes your code more readable.

File: script.js [hunk 8]
```diff
@@ -0,0 +1,201 @@
...
+        <div class="bubble bot-bubble">
+          <p>${message}</p>
+        </div>
+      </section>
+    `;
+  }
+  chat.scrollTop = chat.scrollHeight;
+};
+
+const nextQuestion = (message) => {
```
Comment (Kaliine) [inline script.js:40]: Super cool that you constructed this next question section using conditional statements. This enhances readability so much.

File: script.js [hunk 9]
```diff
@@ -0,0 +1,201 @@
...
+  }
+};
+
+// Starts here
+const greeting = () => {
+  questionNumber = 1;
+  botReply(`Hello there, What's your name?`);
+};
+
+const animal = (msg) => {
```
Comment (Kaliine) [inline script.js:65]: Cool that you passed on (msg) instead of creating a global variable.

File: script.js [hunk 10]
```diff
@@ -0,0 +1,201 @@
...
+    .getElementById("bears")
+    .addEventListener("click", (animal) => nextQuestion("🐻"));
+};
+
+const showWeather = (animal) => {
+  questionNumber++;
+
+  botReply(`oh good choice ${animal}! Do you like warm ☀️ or cold ❄️ places?`);
+
+  if (`${animal}` === "🐯") {
```
Comment (Kaliine) [inline script.js:96]: Cute idea to use symbols for you conditional statements instead of text.

File: script.js [hunk 11]
```diff
@@ -0,0 +1,201 @@
...
+
+    inputWrapper.innerHTML = `<button id="goOnVaccay" value="goOnVaccay">Bye bye ✈️ I'm going on vaccay</button>
+      `;
+    document
+      .getElementById("goOnVaccay")
+      .addEventListener("click", () => nextQuestion(`swichhhhhh ✈️`));
+  }
+};
+
+const GoTravel = () => {
```
Comment (Kaliine) [inline script.js:184]: Really nice add on to play audio as well.

**Review by Kaliine — COMMENTED** [review]
**Code review by Birgit Nehrwein and Karoline Mann.**
- Normally we would suggest to add comments to you JavaScript code to make it easy to read, especially after you’ve been away from it for a while. Or for other people’s benefit. But in your case, the code is super easy to follow! We think it is the use of conditional statements that makes it very clear of what is going on and you are able to read it from top to bottom, instead of going up and down multiple times.
- The code is very tidy and no duplicants. 
- Nothing is broken, it works great.
- If you want to bring your code to the next level we suggest to have a look at responsiveness at different screen sizes, especially mobile view. 

Great job!

File: style.css (hunk 3 above)
Comment (ghost) [reply style.css:95]: in this case, it blur the background behind it :)

File: style.css (hunk 4 above)
Comment (ghost) [reply style.css:108]: its remove the visibility of the scrollbar but keep the function of it :)

File: style.css (hunk 5 above)
Comment (ghost) [reply style.css:178]: we think is ease in the effect when you hovering over the item. Slow down the change to the hovering state

**Comment by ghost** [comment]
Thank you for your good and constructive feedback Karoline and Birgit!// Jessica & Maria

File: style.css (hunk 3 above)
Comment (nehrwein) [reply style.css:95]: Thanks for the reply. Just to let you know - this effect doesn't work in firefox.

File: style.css (hunk 4 above)
Comment (nehrwein) [reply style.css:108]: same here. the scrollbar is still visible in firefox

**Pull request closed without merging**